### PR TITLE
Focus Landscape orientation

### LINKED ISSUE #473: Landscape Orientation for Focus Home Screen
**Actual Behaviour.**  
Creating a Landscape orientation for focus Home screen.

**Expected Behaviour**  
 Should work properly in Landscape mode

**Steps to reproduce it**  
Similar to the portrait mode.

**Additional context**  
NA

**Would you like to work on the issue?**
Issue for Codeheat.

### PR DESCRIPTION
**Pull request by kshitiz-kumar**
Fixes #473 

**Changes**: Added the Focus Landscape Orientation

**Screenshot/s for the changes**:

<image src="https://user-images.githubusercontent.com/38812752/65700030-97ee2f00-e09c-11e9-83c0-36f8a6bf4627.jpeg" height=200>
<br>
>>

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-debug.zip](https://github.com/fossasia/neurolab-android/files/3658498/app-debug.zip)

### CODE REVIEW

**Comment by kshitiz-kumar** [comment]
@jddeep why is that I am getting negative result for Codacy/PR quality review.

File: app/src/main/res/layout-land/activity_focus_parent.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    <LinearLayout
+        android:id="@+id/focus_parent_home_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="200dp"
```
Comment (jddeep) [inline app/src/main/res/layout-land/activity_focus_parent.xml]: Remove hardcoding.

File: app/src/main/res/layout-land/activity_focus_parent.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jddeep) [inline app/src/main/res/layout-land/activity_focus_parent.xml:1]: Rearrange and reformat the code. Remove hardcoding and unnecessary extra blank lines.

**Review by jddeep — CHANGES_REQUESTED** [review]

**Comment by kshitiz-kumar** [comment]
@jddeep review please.

**Comment by kshitiz-kumar** [comment]
@jddeep please review this.

**Review by jddeep — APPROVED** [review]

**Pull request merged**